packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -26,7 +26,7 @@
 #include <cstdlib>
 #include <random>
 
-#if defined( HAVE_DTK_BOOST ) && defined( KOKKOS_ENABLE_SERIAL )
+#if defined( KOKKOS_ENABLE_SERIAL )
```

Comment:
Is it actually possible to remove DTK check in this PR? `BoostOrg` is still an optional dependency in tests/examples, and the tests are not guarded by it within CMakeLists.

Reply:
You cannot run the example without Boost.Program_options

Reply:
Right, but it would simply not compile then if you enable DTK without boost? Just making sure, I understand that we are going to switch soon.

Reply:
See https://github.com/ORNL-CEES/DataTransferKit/pull/550#issuecomment-480665483